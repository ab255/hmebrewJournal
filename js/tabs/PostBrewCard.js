import React, {Component} from 'react';
import {
  ScrollView,
  Text,
  View,
} from 'react-native';

class PostBrewCard extends Component {
  constructor(props) {
    super()
  }

  render() {
    return (
      <ScrollView>
        <View>
          <Text>Fermentation Start</Text>
          <Text>{this.props.dateStartFermentation}</Text>
        </View>
        <View>
          <Text>Fermentation Notes</Text>
          <Text>{this.props.fermentationNotes}</Text>
        </View>
        <View>
          <Text>Packaging Date</Text>
          <Text>{this.props.packagingDate}</Text>
        </View>
        <View>
          <Text>Final Gravity</Text>
          <Text>{this.props.hydrometerReading}</Text>
        </View>
        <View>
          <Text>Type of Packaging</Text>
          <Text>{this.props.typeOfPackaging}</Text>
        </View>
        <View>
          <Text>Packaging Notes</Text>
          <Text>{this.props.notesAboutPackaging}</Text>
        </View>
      </ScrollView>
    )
  }
}

export default PostBrewCard;
