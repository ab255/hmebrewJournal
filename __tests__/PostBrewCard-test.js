import 'react-native';
import React from 'react';
import PostBrewCard from '../js/tabs/PostBrewCard.js';

import renderer from 'react-test-renderer';

it('renders correctly', () => {
  const tree = renderer.create(
    <PostBrewCard />
  ).toJSON();
  expect(tree).toMatchSnapshot();
});
