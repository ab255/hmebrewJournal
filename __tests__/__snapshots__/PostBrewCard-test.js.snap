exports[`test renders correctly 1`] = `
<ScrollView
  style={
    Object {
      "padding": 10,
    }
  }>
  <View>
    <Text
      accessible={true}
      allowFontScaling={true}
      ellipsizeMode="tail"
      style={
        Object {
          "color": "#911F27",
          "fontSize": 20,
          "fontWeight": "bold",
          "marginBottom": 7,
          "textDecorationLine": "underline",
        }
      }>
      Fermentation Start
    </Text>
    <Text
      accessible={true}
      allowFontScaling={true}
      ellipsizeMode="tail"
      style={
        Object {
          "color": "#630A10",
          "fontSize": 16,
          "marginBottom": 10,
        }
      } />
  </View>
  <View>
    <Text
      accessible={true}
      allowFontScaling={true}
      ellipsizeMode="tail"
      style={
        Object {
          "color": "#911F27",
          "fontSize": 20,
          "fontWeight": "bold",
          "marginBottom": 7,
          "textDecorationLine": "underline",
        }
      }>
      Fermentation Notes
    </Text>
    <Text
      accessible={true}
      allowFontScaling={true}
      ellipsizeMode="tail"
      style={
        Object {
          "color": "#630A10",
          "fontSize": 16,
          "marginBottom": 10,
        }
      } />
  </View>
  <View>
    <Text
      accessible={true}
      allowFontScaling={true}
      ellipsizeMode="tail"
      style={
        Object {
          "color": "#911F27",
          "fontSize": 20,
          "fontWeight": "bold",
          "marginBottom": 7,
          "textDecorationLine": "underline",
        }
      }>
      Packaging Date
    </Text>
    <Text
      accessible={true}
      allowFontScaling={true}
      ellipsizeMode="tail"
      style={
        Object {
          "color": "#630A10",
          "fontSize": 16,
          "marginBottom": 10,
        }
      } />
  </View>
  <View>
    <Text
      accessible={true}
      allowFontScaling={true}
      ellipsizeMode="tail"
      style={
        Object {
          "color": "#911F27",
          "fontSize": 20,
          "fontWeight": "bold",
          "marginBottom": 7,
          "textDecorationLine": "underline",
        }
      }>
      Final Gravity
    </Text>
    <Text
      accessible={true}
      allowFontScaling={true}
      ellipsizeMode="tail"
      style={
        Object {
          "color": "#630A10",
          "fontSize": 16,
          "marginBottom": 10,
        }
      } />
  </View>
  <View>
    <Text
      accessible={true}
      allowFontScaling={true}
      ellipsizeMode="tail"
      style={
        Object {
          "color": "#911F27",
          "fontSize": 20,
          "fontWeight": "bold",
          "marginBottom": 7,
          "textDecorationLine": "underline",
        }
      }>
      ABV
    </Text>
    <Text
      accessible={true}
      allowFontScaling={true}
      ellipsizeMode="tail"
      style={
        Object {
          "color": "#630A10",
          "fontSize": 16,
          "marginBottom": 10,
        }
      }>
      0.0
      %
    </Text>
  </View>
  <View>
    <Text
      accessible={true}
      allowFontScaling={true}
      ellipsizeMode="tail"
      style={
        Object {
          "color": "#911F27",
          "fontSize": 20,
          "fontWeight": "bold",
          "marginBottom": 7,
          "textDecorationLine": "underline",
        }
      }>
      Attenuation
    </Text>
    <Text
      accessible={true}
      allowFontScaling={true}
      ellipsizeMode="tail"
      style={
        Object {
          "color": "#630A10",
          "fontSize": 16,
          "marginBottom": 10,
        }
      }>
      0.0%
    </Text>
  </View>
  <View>
    <Text
      accessible={true}
      allowFontScaling={true}
      ellipsizeMode="tail"
      style={
        Object {
          "color": "#911F27",
          "fontSize": 20,
          "fontWeight": "bold",
          "marginBottom": 7,
          "textDecorationLine": "underline",
        }
      }>
      Type of Packaging
    </Text>
    <Text
      accessible={true}
      allowFontScaling={true}
      ellipsizeMode="tail"
      style={
        Object {
          "color": "#630A10",
          "fontSize": 16,
          "marginBottom": 10,
        }
      } />
  </View>
  <View>
    <Text
      accessible={true}
      allowFontScaling={true}
      ellipsizeMode="tail"
      style={
        Object {
          "color": "#911F27",
          "fontSize": 20,
          "fontWeight": "bold",
          "marginBottom": 7,
          "textDecorationLine": "underline",
        }
      }>
      Packaging Notes
    </Text>
    <Text
      accessible={true}
      allowFontScaling={true}
      ellipsizeMode="tail"
      style={
        Object {
          "color": "#630A10",
          "fontSize": 16,
          "marginBottom": 10,
        }
      } />
  </View>
</ScrollView>
`;
